src/custom/pages/Claim/ClaimingStatus.tsx

```diff
@@ -44,31 +58,58 @@ export default function ClaimingStatus() {
           <CowProtocolLogo size={100} />
         )}
       </ConfirmedIcon>
-      <h3>{isConfirmed ? 'Claimed!' : 'Claiming'}</h3>
+
+      <h3>{isConfirmed ? 'Claim successful!' : 'Claiming'}</h3>
       {!isConfirmed && <Trans>{claimedAmount} vCOW</Trans>}
 
       {isConfirmed && (
         <>
           <Trans>
-            <h3>You have successfully claimed</h3>
-          </Trans>
-          <Trans>
-            <p>{claimedAmount} vCOW</p>
-          </Trans>
-          <Trans>
-            <span role="img" aria-label="party-hat">
-              🎉🐮{' '}
-            </span>
-            <p>Welcome to the COWmunnity! :)</p>
+            <h4>
+              Congratulations on claiming <b>{claimedAmount} vCOW!</b>{' '}
+              {isSelfClaiming && <AddToMetamask currency={currency} />}
+            </h4>
+            <p>
+              <span role="img" aria-label="party-hat">
+                🎉🐮{' '}
+              </span>
+              Welcome to the COWmunity! We encourage you to share on Twitter and join the community on Discord to get
+              involved in governance.
+            </p>
           </Trans>
-          {isSelfClaiming ? (
-            <Trans>
-              <p>
-                You can see your vCOW balance in the <Link to="/profile">Profile</Link>
-              </p>
-              <AddToMetamask currency={currency} />
-            </Trans>
-          ) : (
+
+          <BannersWrapper>
+            <Link to="/profile">
+              <SuccessBanner type={'Profile'}>
+                <span>
+                  <Trans>View vCOW balance</Trans>
+                </span>
+                <SVG src={CowProtocolIcon} description="Profile" />
+              </SuccessBanner>
+            </Link>
+            <a
```

Comment:
<ExternalLink> (imported from 'theme') didn't work for me. Kept showing TS errors.